gamemodes/terrortown/entities/entities/ttt_logic_role.lua

```diff
@@ -8,7 +8,7 @@ ENT.Base = "base_point"
 
 local IsValid = IsValid
 
-local ROLE_ANY = 3
+ROLE_ANY = ROLE_ANY or 3
```

Comment:
Is this even needed? Isn't the shared init file loaded before the entity?

Reply:
Nope, tge entity is loaded before the shared file as it seems